### PR TITLE
fix: forward `stdin` from shorebird to flutter process

### LINKED ISSUE #155: `shorebird run` shows output from `flutter run` but doesn't forward input
This may be the correct behavior, but it was confusing to at least one trusted tester.

We show the output from `flutter run` that talks about 'h', 'c', 'q', but if you type those they don't work.

image

I think we would need to listen to stdin and forward to the process if we wanted them to:
https://github.com/shorebirdtech/shorebird/blob/main/packages/shorebird_cli/lib/src/commands/run_command.dart#L53

### PR DESCRIPTION
**Pull request by DarthBenro008**
## Description

- Fixes #155 
- Adds a `dart:io` dependency import to enable usage of `stdin` of the main proccess, so that the input can be captured and forwarded

I was unsure how to add tests, any help would be appreciated, but i have attached a snippet below as part of my testing :D

ezgif com-video-to-gif

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [x] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: packages/shorebird_cli/lib/src/commands/run_command.dart [hunk 1]
```diff
@@ -73,6 +74,13 @@ class RunCommand extends ShorebirdCommand
       logger.err(utf8.decode(event));
     });
 
+    stdin
```
Comment (felangel) [inline packages/shorebird_cli/lib/src/commands/run_command.dart]: Is there any issue with just piping directly?

```dart
flutter.stdin.addStream(stdin);
```

Comment (DarthBenro008) [reply packages/shorebird_cli/lib/src/commands/run_command.dart]: the `addStream()` returns a future and is a blocking process (requires await). I wasn't sure if blocking threads would be the right approach, hence took a non-blocking approach.

Comment (felangel) [reply packages/shorebird_cli/lib/src/commands/run_command.dart]: I think we can wrap it in an unawaited. It seems unnecessary to transform and decode everything imo.

Comment (DarthBenro008) [reply packages/shorebird_cli/lib/src/commands/run_command.dart]: Makes sense! I have updated the commit!

**Review by felangel — APPROVED** [review]

**Pull request merged**